src/open_company_web/api.cljs

```diff
@@ -41,39 +41,39 @@
       (let [body (if (:success response) (json->cljs (:body response)) {})]
         (flux/dispatch dispatcher/companies body)))))
 
-(defn get-company [ticker]
+(defn get-company [symbol]
```

Comment:
ticker is probably better.

"symbol" is a clojure core function: https://clojuredocs.org/clojure.core/symbol